examples/basic-typescript/src/index.tsx

```diff
@@ -26,13 +26,10 @@ type Post = {
 };
 
 function usePosts() {
-  return useQuery(
+  return useQuery<Array<Post>>(
```

Comment:
not really a fan of annotating call-side generics. This should be inferred, should it not?

Reply:
Yeah it is inferred automatically, I was just testing something, that's why added it, forgot to remove it.